control/reset.py

```diff
@@ -25,15 +26,18 @@ def createReset():
                             mongodb_settings.get('port'))
     rabbitmq_conn = RabbitMQConn()
 
-    return Reset(mongo_conn, rabbitmq_conn)
+    connman = ConnmanClient()
```

Comment:
Considering that ConnmanClient() may not be successfully initialized, shouldn't this be handled here?

Maybe handling an Exception would be appropriated.